docs/markdown/Fs-module.md

```diff
@@ -34,6 +35,23 @@ by the string is a symbolic link.
 
 ## File Parameters
 
+### is_absolute
+
+Return a boolean indicating if the path string specified is absolute, WITHOUT expanding `~`.
+
+Examples:
+
+```meson
+fs.is_absolute('~')   # false unless you literally have a path with string name `~`
```

Comment:
How does the latter part tie into this? Even if you have a file named `~`, the path `~` would not be absolute. `/foo/bar/~` would be.

Reply:
oops, thanks